tests/xtimer_usleep/README.md

```diff
@@ -1,8 +1,10 @@
 # xtimer_usleep test application
 
-This test tests `xtimer_usleep()` both against the timings of
-`xtimer_now_usec()` and by providing capabilities to compare against an external
-timer.
+This test tests `xtimer_usleep()` both against the timings of `xtimer_now_usec()`
+ and by providing capabilities to compare against an external timer.
+
+The sleep times can be probed at a pin if SLEEP_PIN` is set to 1 and the respective
+gpio pin is define as `SLEEP_GPIO_PIN`.
```

Comment:
You missed a ` before SLEEP_PIN. You might give more hints on how to use it, e.g. add that it can be probed with an scope or something like that.